tests/tools/tflite_comparator/src/tflite_comparator.cc

```diff
@@ -119,18 +119,50 @@ inline size_t sizeOfNnfwType(NNFW_TYPE type)
 }
 
 template <typename T>
-bool compareBuffersExact(const T *ref_buf, const std::vector<uint8_t> &act_buf, uint32_t index)
+bool compareBuffers(const T *ref_buf, const std::vector<uint8_t> &act_buf, uint32_t index)
```

Comment:
Q. Is it possible that this function is used for exact match between int (not for quantized value) ?

Reply:
Yes. It's possible.

Reply:
Is there a chance to pass the mismatch between Integers due to non-zero tolerance ?

Reply:
Not now. But if we uses TFLite 2.8, test model file `reduce_max_uint8.tflite` (`tests/scripts/models/tflite/reduce_sum/uint8/config.sh`) return diff 1 on some output elements. It may be related with kernel implementation.

```
Run reduce_sum/uint8
======================
[Execution] Stage start!
[Execution] Model is deserialized!
[Execution] Model compiled!
[Execution] Input data is defined!
[Execution] Done!
[Comparison] Stage start!
[Comparison] TFLite run done!
Output #0, Element Index : 0, ref: G, act: H (diff: 1)
Output #0, Element Index : 2, ref: �, act: � (diff: 1)
Output #0, Element Index : 4, ref: �, act: � (diff: 1)
Output #0, Element Index : 5, ref: , act:  (diff: 1)
Output #0, Element Index : 7, ref: �, act: � (diff: 1)
Output #0, Element Index : 8, ref: 3, act: 4 (diff: 1)
Output #0, Element Index : 9, ref: r, act: s (diff: 1)
Output #0, Element Index : 10, ref: �, act: � (diff: 1)
Output #0, Element Index : 11, ref: B, act: C (diff: 1)
Output #0, Element Index : 12, ref: �, act: � (diff: 1)
Output #0, Element Index : 13, ref: K, act: L (diff: 1)
Output #0, Element Index : 14, ref: F, act: G (diff: 1)
Output #0, Element Index : 15, ref: �, act: � (diff: 1)
Output #0, Element Index : 17, ref: �, act: � (diff: 1)
Output #0, Element Index : 18, ref: �, act:  (diff: 1)
Output #0, Element Index : 19, ref: R, act: S (diff: 1)
Output #0, Element Index : 20, ref: �, act: � (diff: 1)
[Comparison] outputs is not equal!
[Comparison] Done!
```